index.js

```diff
@@ -1,51 +1,66 @@
-let Service;
-let Characteristic;
 const ApcAccess = require('apcaccess');
 const { logOnlyError, logMin } = require('./lib/logUpdate');
 
+const DEFAULT_INTERVAL = 1;
+const DEFAULT_NAME = 'APC UPS';
+const DEFAULT_MANIFACTURER = 'American Power Conversion';
+const DEFAULT_MODEL = 'APCAccess UPS';
+const DEFAULT_PORT = '3551';
+
+const FULLY_CHARGED = 100;
+const SECOND = 1000;
+const UNKOWN = 'unkown';
```

Comment:
nit/typo: `UNKNOWN / unknown`

Reply:
Dyslexia stricts again. I Will fix in a bit